notification-channel-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_channel.proto

```diff
@@ -13,8 +13,22 @@ message NotificationChannelMutableData {
   string channel_name = 1;
   repeated EmailChannelConfig email_channel_config = 2;
   repeated WebhookChannelConfig webhook_channel_config = 3;
+  repeated AwsS3BucketChannelConfig bucket_channel_config = 4;
```

Comment:
it took me 3 or 4 reads (Sorry!) but I just noticed the name is generic ("bucket"), but the type is s3-specific. If we wanted to add support for GCS next, what would that look like? I think another top level field, so maybe put that specific into this field name too.

Reply:
Sure, adding